### PR TITLE
Remove packagefile match check to fix Git packages

### PR DESCRIPTION
**Pull request by not-my-segfault**
This check ends up breaking every `-git` package

If a `PKGBUILD` were to potentially install malware in the form of an extra `.pkg.tar.zst` onto a user's computer, they will be able to see by reviewing the `PKGBUILD`

Any potential input welcome, of course

### CODE REVIEW

**Comment by Trivernis** [comment]
Hm without the check we'd run into the issue that everything a PKGBUILD produces gets installed.
You can test that with the clion package

File: src/operations/aur_install/common.rs [hunk 1]
```diff
@@ -196,25 +201,32 @@ async fn build_package(
...
-    });
-    tracing::debug!("Archives: {packages:?}");
+    let archives = MakePkgBuilder::package_list(build_path).await?;
+    tracing::debug!("Archives: {archives:?}");
+
+    let mut pkgs_produced: HashMap<String, PathBuf> = HashMap::new();
+    let alpm = crate::internal::alpm::get_handler()?;
+
+    for ar in archives {
+        let pkg = alpm
```
Comment (Trivernis) [inline src/operations/aur_install/common.rs:211]: Can you encapsulate alpm stuff in a different module?

File: src/operations/aur_install/common.rs [hunk 2]
```diff
@@ -196,25 +201,32 @@ async fn build_package(
...
-            .starts_with(&match_name)
-    });
-    tracing::debug!("Archives: {packages:?}");
+    let archives = MakePkgBuilder::package_list(build_path).await?;
+    tracing::debug!("Archives: {archives:?}");
+
+    let mut pkgs_produced: HashMap<String, PathBuf> = HashMap::new();
+    let alpm = crate::internal::alpm::get_handler()?;
+
+    for ar in archives {
```
Comment (Trivernis) [inline src/operations/aur_install/common.rs:210]: lazy with variable naming again?

**Review by Trivernis — CHANGES_REQUESTED** [review]

File: src/operations/aur_install/common.rs (hunk 1 above)
Comment (not-my-segfault) [reply src/operations/aur_install/common.rs:211]: I'll split it off into the already existing `alpm` module we have

Comment (not-my-segfault) [reply src/operations/aur_install/common.rs]: This should be good :+1:

File: src/operations/aur_install/common.rs (hunk 2 above)
Comment (not-my-segfault) [reply src/operations/aur_install/common.rs:210]: Alright, I'll add this to contributing guidelines too just to make sure I don't do something like this again lol

Comment (Trivernis) [reply src/operations/aur_install/common.rs:210]: Lol

**Review by Trivernis — APPROVED** [review]

**Pull request merged**